src/handlers/extensions/helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { mutate } from '~amplifyClient';
+import { ExtensionParams } from '~types';
+import { getVotingClient } from '~utils';
+
+const getVotingReputationParams = async (
+  colonyAddress: string,
+): Promise<ExtensionParams> => {
+  const votingClient = await getVotingClient(colonyAddress);
+  const totalStakeFraction = (
+    await votingClient.getTotalStakeFraction()
+  ).toString();
+  const voterRewardFraction = (
+    await votingClient.getVoterRewardFraction()
+  ).toString();
+  const userMinStakeFraction = (
+    await votingClient.getUserMinStakeFraction()
+  ).toString();
+  const maxVoteFraction = (await votingClient.getMaxVoteFraction()).toString();
+  const stakePeriod = (await votingClient.getStakePeriod()).toString();
+  const submitPeriod = (await votingClient.getSubmitPeriod()).toString();
+  const revealPeriod = (await votingClient.getRevealPeriod()).toString();
+  const escalationPeriod = (
+    await votingClient.getEscalationPeriod()
+  ).toString();
+
+  return {
+    votingReputation: {
+      totalStakeFraction,
+      voterRewardFraction,
+      userMinStakeFraction,
+      maxVoteFraction,
+      stakePeriod,
+      submitPeriod,
+      revealPeriod,
+      escalationPeriod,
+    },
+  };
+};
+
+export const addVotingReputationParamsToDB = async (
+  extensionAddress: string,
+  colonyAddress: string,
+): Promise<void> => {
+  await mutate('updateColonyExtensionByAddress', {
+    input: {
+      id: extensionAddress,
+      params: await getVotingReputationParams(colonyAddress),
```

Comment:
I'm not a fan of this pattern personally. I'd have this assigned to a variable then pass that along to the mutation input.